workspaces/orchestrator/plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -246,12 +246,9 @@ export async function createBackendRouter(
       throw new Error('next is undefined');
     }
 
-    return routerApi.openApiBackend.handleRequest(
-      req as Request,
-      req,
-      res,
-      next,
-    );
+    return routerApi.openApiBackend
+      .handleRequest(req as Request, req, res, next)
+      .catch(next);
```

Comment:
I was thinking of this approach at a certain point, but it's missing the audit log

Reply:
Can you add the auditLog to here somehow?

Reply:
Added. This is just the last-option handler, a generic one. We should keep using try{} blocks inside as we do so far, so we can get more precise handling. This top-level handler is for left-overs only.


The log newly contains:
```
➤ YN0000: [backend]: 2024-12-05T09:25:02.414Z permission info user:development/guest is DENY for permission 'orchestrator.workflow' and action 'read' actor=[object Object] meta=[object Object] request=undefined isAuditLog=true response=undefined eventName=PermissionEvaluationCompleted stage=evaluatePermissionAccess status=succeeded
➤ YN0000: [backend]: 2024-12-05T09:25:02.415Z rootHttpRouter info [05/Dec/2024:09:25:02 +0000] "POST /api/permission/authorize HTTP/1.1" 200 107 "-" "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" type=incomingRequest
➤ YN0000: [backend]: 2024-12-05T09:25:02.416Z rootHttpRouter info [05/Dec/2024:09:25:02 +0000] "GET /api/catalog/entities?fields=metadata.name,metadata.namespace,spec.parent&filter=kind%3DGroup HTTP/1.1" 200 95 "-" "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" type=incomingRequest
➤ YN0000: [backend]: 2024-12-05T09:25:02.417Z permission info user:development/guest is DENY for permission 'orchestrator.workflow.wait-or-error' and action 'read' actor=[object Object] meta=[object Object] request=undefined isAuditLog=true response=undefined eventName=PermissionEvaluationCompleted stage=evaluatePermissionAccess status=succeeded
➤ YN0000: [backend]: 2024-12-05T09:25:02.417Z rootHttpRouter info [05/Dec/2024:09:25:02 +0000] "POST /api/permission/authorize HTTP/1.1" 200 107 "-" "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)" type=incomingRequest
➤ YN0000: [backend]: 2024-12-05T09:25:02.418Z orchestrator error Not authorize to request the /v2/workflows/wait-or-error/overview endpoint actor=[object Object] meta=[object Object] request=[object Object] isAuditLog=true response=[object Object] eventName=getWorkflowOverviewByIdEndpointHit stage=authorization status=failed errors=[object Object]
➤ YN0000: [backend]: 2024-12-05T09:25:02.418Z orchestrator error Exception thrown during processing request /v2/workflows/wait-or-error/overview , Unauthorized actor=[object Object] meta=[object Object] request=[object Object] isAuditLog=true response=undefined eventName=genericErrorHandler stage=completion status=failed errors=[object Object]
```


![Screenshot From 2024-12-05 10-13-09](https://github.com/user-attachments/assets/49e6bb01-66e5-4245-8eb2-e8e37e9306c0)